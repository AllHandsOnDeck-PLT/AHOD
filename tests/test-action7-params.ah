when do string A(string x,int y):
{
    string z
    z = x
    do PRINT(y)
    return z 
}
main:
{
    string w
    w = do A("Ahoy Matey, to AHOD", 3) 
    do PRINT(w)
}
