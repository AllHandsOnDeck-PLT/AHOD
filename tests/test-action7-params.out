3
Ahoy Matey, to AHOD